docs/Guides/Detecting-When-Clients-Abort.md

```diff
@@ -88,19 +82,17 @@ Our code is setting up a Fastify server which includes the following
 functionality:
 
 - Accepting requests at http://localhost:3000, with a 3 second delayed response 
-of { ok: true }.
+of `{ ok: true }`.
 - An onRequest hook that triggers when every request is received.
 - Logic that triggers in the hook when the request is closed.
-- Logging that occurs when the closed request attribute 'aborted' is true.
+- Logging that occurs when the closed request property `aborted` is true.
 
 In the request close event, you should examine the diff between a successful 
-request and one aborted by the client to determine the best attribute for your 
-use case. There are many other attributes on a request that will differ between 
-a successfully closed request and one that has been aborted by the client. 
-Examples of such attributes include:
-
-- destroyed
-- errors
+request and one aborted by the client to determine the best property for your 
+use case. There are many other properties on a request that will differ between 
```

Comment:
I wouldn't say that there are "many". There are a few yeah, but not many

Reply:
Got it. There were at least 10 if I recall correctly but only a few that users might use for this type of thing. I think it's ok to just remove that sentence.